### ISSUE TITLE
Accessibility: A11y_Live Chat Widget _Chat Window_ Screenreader: After sending any message screen reader announce same information many times.

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3055]
Copied from #1750244.

[1750244_A11y_Live Chat Widget _Chat Window_ Screenreader.zip](https://github.com/microsoft/BotFramework-WebChat/files/4397036/1750244_A11y_Live.Chat.Widget._Chat.Window_.Screenreader.zip)

“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.” 

 

Environment Details:

Application Name: Live Chat Widget

#URL: https://xxx.blob.core.windows.net/portals/democ2occatest4.html

Browser Versions: Microsoft Edge Version 80.0.361.69 (Official build) (64-bit)

Windows Version: Windows10

Chrome Version: Version 80.0.3987.149 (Official Build) (64-bit)

 

 

Repro Steps: ​​

1)Launch the application url : https://xxx.blob.core.windows.net/portals/democ2occatest4.html

2)welcome to live chat support Screen get open.

4)Navigate to "Chat Now button" and click on it.

5)"Chat Form" screen get open.

6)Fill all the mandatory detail and then Click on "Submit" Button

7)now "Chat Window" get open.

8)Type your message and click on send button.

Actual:

 

After sending any message screen reader announce same information many times.

Same issue is observed with edge+narrator.

 

Same issue is observed for all the Chat Window Screens.

Expected:

Information should be announcing only single time.

User Impact:

Screenreader users will face difficulty as screen reader unnecessarily announces same information many times.

Recommendations: ​

Refer below link which is repository of bug fixes code snippets:

https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

 

MAS Reference:

MAS1.3.1:

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by compulim** [comment]
Repeated narration of timestamp is fixed in PR #3076.

I think this is dupe of #3052
